src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -178,6 +180,18 @@ public void setAngleSpeed(double Speed) {
 		launcherAnglePIDController.setReference(Speed, ControlType.kPosition);
 	}
 
+	public boolean atTargetSpeed() {
+
+		return MathUtil.isNear(
```

Comment:
With the ability to run independently, seems like it should be checking both motor speeds, especially for shooting into amp or just shot into amp and now shooting into speaker?
Is that the right tolerance, 0.1?